scripts/devops_tasks/test_regression.py

```diff
@@ -23,7 +23,8 @@
     find_packages_missing_on_pypi,
     find_whl,
     find_tools_packages,
-    get_installed_packages
+    get_installed_packages,
+    extend_dev_requirements
```

Comment:
I don't see this used anywhere or am i overlooking something.

Reply:
👍